web/styles/popovers.css

```diff
@@ -735,6 +735,17 @@ ul {
     left: -1px;
 }
 
+.giphy-popover {
+    .tippy-box {
+        border-bottom-left-radius: 7px;
+        border-bottom-right-radius: 7px;
+    }
+
+    .tippy-content {
+        padding: 0;
+    }
+}
```

Comment:
A comment explaining why we need this custom styling would be helpful.

Reply:
Added!